### ISSUE TITLE
Use .tfvars for Terraform variables

### ISSUE THREAD

**Issue by aknot242** [f5devcentral/xcedgedemoguide#2]
Terraform recommends [using a .tfvars file for variable values](https://developer.hashicorp.com/terraform/tutorials/configuration-language/variables#assign-values-with-a-file) rather than updating the variables.tf file. 

An additional concern with the existing implementation is that someone contributing to your repo (or a fork of it) may accidentally commit sensitive information in the `var.tf` file, as this file is already tracked and versioned (it is not in the `.gitignore` list).

**Comment by yoctoalex** [comment]
Good point! We'll update the instructions to use .tfvars instead and add it to the .gitignore file. Thank you!
